cmake/std/PullRequestLinuxDriverTest.py

```diff
@@ -42,706 +40,221 @@
+                          help='Repo with the new changes',
+                          required=True)
+
+    required.add_argument('--sourceBranch',
+                          dest="sourceBranch",
+                          action='store',
+                          help='Branch with the new changes',
+                          required=True)
+
+    required.add_argument('--targetRepo',
+                          dest="targetRepo",
+                          action='store',
+                          help='Repo to merge into',
+                          required=True)
+
+    required.add_argument('--targetBranch',
+                          dest="targetBranch",
+                          action='store',
+                          help='Branch to merge into',
+                          required=True)
+
+    required.add_argument('--job_base_name',
+                          dest="job_base_name",
+                          action='store',
+                          help='The Jenkins job base name',
+                          required=True)
+
+    required.add_argument('--github_pr_number',
+                          dest="github_pr_number",
+                          action='store',
+                          help='The github PR number',
+                          required=True)
+
+    required.add_argument('--job_number',
+                          dest="job_number",
+                          action='store',
+                          help='The Jenkins build number',
+                          required=True)
+
+    optional.add_argument('--config',
+                          dest='configfile',
+                          action='store',
+                          default=os.path.join(cwd, "pr_config/pullrequest.ini"),
+                          help="Configuration file. Default=%(default)s",
+                          required=False)
+
+    optional.add_argument('--workspaceDir',
+                          dest="workspaceDir",
+                          action='store',
+                          default=default_workspace,
+                          help="The local workspace directory that Jenkins set up." +
+                               " Default={}".format(default_workspace),
+                          required=False)
+
+    optional.add_argument('--packageEnables',
+                          dest="package_enables",
+                          action="store",
+                          default=default_package_enables,
+                          help="Custom packageEnables.cmake override." +
+                               " Default={}".format(default_package_enables))
+
+    optional.add_argument('--subprojects_file',
+                          dest="subprojects_file",
+                          action="store",
+                          default=default_subprojects_file,
+                          help="Custom subprojects file." +
+                               " Default={}".format(default_subprojects_file))
```

Comment:
Not sure what this means.

Reply:
It's an option normal users probably should never touch.  the subprojects file is a generated file that gets passed into the `ctest` call that ultimately drives everything.